components/SignInForm.tsx

```diff
@@ -0,0 +1,150 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { useRef, useState } from "react";
+
+// Logo Import
+import logo from "../public/Cacart_logo.png";
+import googlelogo from "../public/google_logo.png";
+
+type FormInputs = {
+  email: string;
+  password: string;
+};
+
+export default function SignInForm() {
+  // Set userData
+  const userData = useRef<FormInputs>({
+    email: "",
+    password: "",
+  });
+
+  const validateEmail = (email: string) => {
```

Comment:
We should add comments to these constants declared.

Reply:
I've applied your suggestion in the latest commit. Thank you for the advice!